src/theory/arith/cut_log.cpp

```diff
@@ -106,13 +106,11 @@ CutInfo::CutInfo(CutInfoKlass kl, int eid, int o)
   , d_cutVec()
   , d_mAtCreation(-1)
   , d_rowId(-1)
-  , d_exactPrecision(NULL)
-  , d_explanation(NULL)
+  , d_exactPrecision(nullptr)
```

Comment:
Can these two initialisations be omitted after smart pointers are applied for the class members?

Reply:
They probably can, although I'm not sure. @mpreiner  do you know?

Reply:
You could, but I think until now we always explicitly initialized them.

Reply:
How do you think about to pick any further opportunities up for source code reduction because of smart pointer usage?